### PR TITLE
[Issue Fix] Task: Remove Site Switcher in the Editor

### LINKED ISSUE #14744: Remove Site Switcher in the Editor
This feature is a source of issues for developers and a source of confusion for our users. We're aligning with the Android app and dropping the feature.

When the feature is removed, please close the following tickets (issues related to the site switcher):
- https://github.com/wordpress-mobile/WordPress-iOS/issues/7755
- possibly this one too https://github.com/wordpress-mobile/gutenberg-mobile/issues/3171

### PR DESCRIPTION
**Pull request by shahrukhalam**
# Fixes
- Fixes #14744

# Description
We have decided to remove **Site Switcher** in the Editor after collecting the Analytics Data as it's not a widely used feature & more importantly it was a source of few inconsistencies as captured on this (link on _Internal References_ Section) proposal.
This PR addresses that.

# Testing instructions
- Start a new post in Gutenberg or Aztec Editor
- We shouldn't have Site Switcher
- Instead we should have the _Site Title_ as the ViewController Title
- During intermediate states like `Generating Preview` & `Media Uploading`, when we show left bar button item(s), _Site Title_ should hide

# Screenshots

|Before|After (Gutenberg)|After (Aztec)|After (Intermediate state of _Generating Preview_ in Gutenberg)|
|---|---|---|---|
|Before|After new Gutenberg|After new Aztec|Simulator Screen Shot - iPhone 12 Pro Max 14 1 - 2021-04-01 at 16 22 59|

# Screen Recording
### Before
https://user-images.githubusercontent.com/16770566/112341452-726cd080-8ce7-11eb-9b52-51e0617e8e5e.mov
### After
https://user-images.githubusercontent.com/16770566/112641457-6e1bf100-8e68-11eb-9638-7872b9e95c75.mov
### After (Intermediate state of _Generating Preview_)
https://user-images.githubusercontent.com/16770566/113284507-5e3c5b00-9307-11eb-88ad-646bbff96af1.mov

# PR submission checklist

- [x] I have considered adding unit tests where possible.

N/A

- [x] I have considered adding accessibility improvements for my changes.

|Before|After|
|---|---|
|Before Accessibility|Screenshot 2021-03-26 at 7 24 39 PM|

**Note:** _Pringles_ in the after image is the `Site Title`

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

## Note
- Big shoutout 💯 🥳  to @ceyhun, my trial buddy for helping me out throughout. 
Please have a 👀, add the `label` & `milestone`, give some ❤️ if you like it 🤝 

## Internal References
For more details, please checkout
- proposal (p9ugOq-1fv-p2)
- my Trial plan (pbMoDN-2et-p2)

### CODE REVIEW

**Comment by ceyhun** [comment]
Thanks for the PR @shahrukhalam 🎉 Seems to me like there's still a lot of potential to remove lots of code while removing the switcher (maybe even some assets like the chevron?). What do you think about pursuing those as well?

> I think we can put the Site Name as the ViewController Title instead of LeftBarButtonItem as it's no longer an action. What do you think.

I agree, semantically it would make more sense and maybe it could bring more opportunities for code removal. But I checked the Android design and it's also left aligned. Do you think it would require a lot of effort to both make it a ViewController title and still have left alignment?

**Comment by shahrukhalam** [comment]
Thanks for the suggestion @ceyhun 💯 
> Seems to me like there's still a lot of potential to remove lots of code while removing the switcher (maybe even some assets like the chevron?)

Yeah, we can clear out quite a lot of things like: entire `PostEditor + BlogPicker`, `NavBarTitleDropdownButton` which uses the _chevron_ asset but App itself is not using it.
I would love to remove them if we think we won't need them in future.

> But I checked the Android design and it's also left aligned. Do you think it would require a lot of effort to both make it a ViewController title and still have left alignment?

Making Title left aligned shouldn't take a lot of effort as we have to play with UILabel's constraints w.r.t TitleView's superview. But, I believe it won't be very pretty, might look like a hack.
Also I think in Android left aligned title is by default. I believe for iOS users, it won't be as intuitive.

**Comment by ceyhun** [comment]
> I would love to remove them if we think we won't need them in future.

Sounds like a good opportunity to me 👍 

> Making Title left aligned shouldn't take a lot of effort as we have to play with UILabel's constraints w.r.t TitleView's superview. But, I believe it won't be very pretty, might look like a hack.
Also I think in Android left aligned title is by default. I believe for iOS users it won't be as intuitive.

Maybe it's best to involve @iamthomasbishop at this stage 😄

**Comment by shahrukhalam** [comment]
> Yea, basically I would just follow the platform standard (centered on iOS, 72dp from the left on Android)

Had a quick chat with @iamthomasbishop & we are happy😄 with centre aligned Navigation Title in iOS

@ceyhun It would give us ample opportunity to remove things 🥳 💯
I ❤️ it 😄

File: WordPress/Classes/ViewRelated/Views/LoadingStatusView.swift [hunk 1]
```diff
@@ -15,7 +15,7 @@ class LoadingStatusView: UIView {
 
     private lazy var titleLabel: UILabel = {
         let label = UILabel()
-        label.textColor = .white
+        label.textColor = .appBarText
```
Comment (shahrukhalam) [inline WordPress/Classes/ViewRelated/Views/LoadingStatusView.swift:16]: `Generating Preview` was not visible in light mode, Please correct me if we are not using the right color
@ceyhun

**Comment by shahrukhalam** [comment]
@ceyhun
- I have updated the approach with `TitleView` at 70ffacfaa8052658e69d7c3cabc8a20c45811931
- Updated the PR description for the same

Please have a look.

**Comment by shahrukhalam** [comment]
@ceyhun I am thinking to do the cleanup/removal in a separate PR to keep the implementation clean in this one & I am on it 🚀 
In this PR, I have removed just the source, which is `blogPickerBarButtonItem`

**Comment by ceyhun** [comment]
> Had a quick chat with @iamthomasbishop & we are happy😄 with centre aligned Navigation Title in iOS

👍 I couldn't find the chat here or on the original issue, I'm guessing it was over Slack DM.




I'm not sure about showing the title on the intermediate state though, seems a bit crowded to me. What do you think? @iamthomasbishop any thoughts on showing the site title in the intermediate states?

File: WordPress/Classes/ViewRelated/Views/LoadingStatusView.swift (hunk 1 above)
Comment (ceyhun) [reply WordPress/Classes/ViewRelated/Views/LoadingStatusView.swift:16]: Good catch! Looks good to me 👍

File: WordPress/Classes/ViewRelated/Views/WPBlogSelectorButton.m [hunk 2]
```diff
@@ -20,7 +20,6 @@ + (instancetype)buttonWithFrame:(CGRect)frame buttonStyle:(WPBlogSelectorButtonS
 
     button.titleLabel.adjustsFontSizeToFitWidth = NO;
     [button setImage:[UIImage imageNamed:@"icon-nav-chevron"] forState:UIControlStateNormal];
-    [button setAccessibilityHint:NSLocalizedString(@"Tap to select which blog to post to", @"This is the blog picker in the editor")];
```
Comment (ceyhun) [inline WordPress/Classes/ViewRelated/Views/WPBlogSelectorButton.m:23]: > @ceyhun I am thinking to do the cleanup/removal in a separate PR to keep the implementation clean in this one & I am on it 🚀
In this PR, I have removed just the source, which is blogPickerBarButtonItem

Sound good to me 👍 Should we maybe leave this change to that PR as well? Or is this related to this one?

File: WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift [hunk 3]
```diff
@@ -905,13 +906,13 @@ class AztecPostViewController: UIViewController, PostEditor {
...
 
-    func reloadBlogPickerButton() {
+    func reloadBlogTitleView() {
         var pickerTitle = post.blog.url ?? String()
         if let blogName = post.blog.settings?.name, blogName.isEmpty == false {
             pickerTitle = blogName
         }
 
-        navigationBarManager.reloadBlogPickerButton(with: pickerTitle, enabled: !isSingleSiteMode)
+        navigationBarManager.reloadBlogTitleView(text: pickerTitle)
```
Comment (ceyhun) [inline WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift]: Maybe we can rename `pickerTitle` as well?

**Comment by shahrukhalam** [comment]
@ceyhun 

> 👍 I couldn't find the chat here or on the original issue, I'm guessing it was over Slack DM.

Yeah, it was over slack DM 😄  

> I'm not sure about showing the title on the intermediate state though, seems a bit crowded to me. What do you think?

Likewise 😄

File: WordPress/Classes/ViewRelated/Views/WPBlogSelectorButton.m (hunk 2 above)
Comment (shahrukhalam) [reply WordPress/Classes/ViewRelated/Views/WPBlogSelectorButton.m:23]: 👍 

> Should we maybe leave this change to that PR as well? Or is this related to this one?

I think it is 😄

I think this line was not really needed, it was getting ignored by the OS as `AccessibilityLabel` of **barButtonItem** was sufficient for the user.

**Refs**
Stackoverflow: [VoiceOver Accessibility accessibilityHint not announced](https://stackoverflow.com/questions/48299040/voiceover-accessibility-accessibilityhint-not-announced)
Apple Documentation: [Accessibilityhint](https://developer.apple.com/documentation/objectivec/nsobject/1615093-accessibilityhint)

File: WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift (hunk 3 above)
Comment (shahrukhalam) [reply WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift]: Fixed at 33aaa72

**Comment by iamthomasbishop** [comment]
@shahrukhalam @ceyhun Agreed this looks off, but as I was testing the current build I just realized this looks a bit different from previous builds, because it seems the "generating preview" message has already been changed (probably with the recent re-work of NavigationBars across WPiOS) to be a centered spinner UI. This is what it looks like:

IMG_2491

I'm not a huge fan of this, and have some ideas for how we could potentially improve the UX, but for for this current project, we might as well just center the Site Title and leave the spinner as-is. What do y'all think?

**Comment by shahrukhalam** [comment]
@iamthomasbishop 

> but as I was testing the current build I just realized this looks a bit different from previous builds, because it seems the "generating preview" message has already been changed (probably with the recent re-work of NavigationBars across WPiOS) to be a centered spinner UI.

Actually it appears like that as:
- `generating preview` label is there, but it's color is **white**.
This PR corrects it to **appBarText**, Please correct me if that's not the right color.

|Current Build|After the above change|
|---|---|
|Before Preview|After Preview|
- `spinner` might appears as centered on smaller devices as `generating preview` text on the left of `spinner` is big. It looks okay on iPhone 12 Pro Max as shown above.

---
> I'm not a huge fan of this, and have some ideas for how we could potentially improve the UX, but for for this current project, we might as well just center the Site Title and leave the spinner as-is. What do y'all think?

I think it's okay to tell us your ideas, we will take it up if we think we can accommodate it in the trial. @ceyhun what do you think.

---
Coming back to the Nav Bar Title, we have three issues:
1. it seems too crowded as pointed out by @ceyhun [here](https://github.com/wordpress-mobile/WordPress-iOS/pull/16158#issuecomment-809445474)

iPhone 12 Pro Max

2. Nav Bar Title will be pushed to the right as shown above because `generating preview` text is big enough. So, it won't be centered. It's the default behaviour in iOS.
3. In smaller screens, Nav Bar Title might get clipped as shown below. It's the default behaviour in iOS.

|iPhone 12 Pro Max|iPhone SE|
|---|---|
|iPhone 12 Pro Max|iPhone SE|

Do you think we should hide the Nav Bar Title in these cases (intermediate states like `generating preview`, `saving draft` etc).

**Comment by iamthomasbishop** [comment]
> Do you think we should hide the Nav Bar Title in these cases (intermediate states like generating preview, saving draft etc).

Yes, we can probably just hide the navbar title 👍

**Comment by shahrukhalam** [comment]
@ceyhun Added show & hide logic for the TitleView at b2ebe0a, updated the PR description, Please have a look

https://user-images.githubusercontent.com/16770566/113283610-1ec13f00-9306-11eb-9534-571b76941f21.mov

@iamthomasbishop Please have a look at ^

**Comment by iamthomasbishop** [comment]
@shahrukhalam That should work, but perhaps we should also center the label/spinner so that the labels don't "jump" from centered to left-aligned and back.

**Comment by shahrukhalam** [comment]
> but perhaps we should also center the label/spinner so that the labels don't "jump" from centered to left-aligned and back.

@iamthomasbishop we could center the `Label + Spinner` as well, but they have Actions behaving as Left Bar Button Items.

**Comment by iamthomasbishop** [comment]
> we could center the Label + Spinner as well, but they have Actions behaving as Left Bar Button Items.

Ah, okay. We can defer that for a future task if it'll require more extensive re-work. 👍

**Comment by shahrukhalam** [comment]
> > we could center the Label + Spinner as well, but they have Actions behaving as Left Bar Button Items.
> 
> Ah, okay. We can defer that for a future task if it'll require more extensive re-work. 👍

@iamthomasbishop I meant `Label + Spinner` is an action, hence it should be a left bar button item as per the standard. So, I strongly feel it shouldn't be centered.

We do have another option
> so that the labels don't "jump" from centered to left-aligned and back.

which is to make _Site Title_ also left aligned, but without any tap action. Does that make sense! Please let me know what do you think.

CC: @ceyhun

**Comment by shahrukhalam** [comment]
> We do have another option
> 
> > so that the labels don't "jump" from centered to left-aligned and back.
> 
> which is to make _Site Title_ also left aligned, but without any tap action. Does that make sense! Please let me know what do you think.

https://user-images.githubusercontent.com/16770566/113610262-7fd07600-966a-11eb-9ec4-c2ece6284d84.mov

@iamthomasbishop This is what I meant, what do you think!

**Comment by iamthomasbishop** [comment]
> what do you think

I definitely preferred the center-aligned title, but if we're unable to center both that and the "Generating..." UI at this time, this feels like a pretty good option because the transition between the two states is pretty smooth.

**Comment by shahrukhalam** [comment]
> I definitely preferred the center-aligned title, but if we're unable to center both that and the "Generating..." UI at this time

We are able to center both that and the "Generating..." UI & we will 😄

---
I was little skeptical about it as `Uploading Media` has an action on classic editor:

https://user-images.githubusercontent.com/16770566/113624897-d6937b00-967d-11eb-94fc-3c61ae52a7a9.mov

Then, I remembered @ceyhun told me earlier that we would be soon deprecating it:



---
Now, I would love to make everything centered & make you happy 😜 
Will put a demo here soon...

**Comment by shahrukhalam** [comment]
> Now, I would love to make everything centered & make you happy 😜 
Will put a demo here soon...

@iamthomasbishop what do you think! ⬇️ 

https://user-images.githubusercontent.com/16770566/113745846-4c055700-9723-11eb-8176-6765dae3138a.mov

**Comment by iamthomasbishop** [comment]
@shahrukh-alam Looks great! I'd be fine shipping this as-is, but I just have one more tiny question regarding the font-weight on the "Generating preview" label. Are we manually setting the font-weight or is that property simply an inherited text style? If we're manually setting a weight, can we change the weight to regular? I think the heavier weight is typically reserved for headings and other emphasized labels — unless I'm missing something, regular-weight would seem more appropriate for this label. // cc @ceyhun

**Comment by shahrukhalam** [comment]
> Are we manually setting the font-weight or is that property simply an inherited text style? If we're manually setting a weight, can we change the weight to regular?

@iamthomasbishop 
Just checked, we are manually setting the font-weight to **Bold** & the size to **14**, I can change it to _regular_ 👍 
Font size is fine, right!

**Comment by iamthomasbishop** [comment]
Sounds good! 

> Font size is fine, right!

I think so. We could go larger, but let's just see how the current font-size (14pt) looks in relation to the Navbar Title.

**Comment by ceyhun** [comment]
> I was little skeptical about it as Uploading Media has an action on classic editor:

https://user-images.githubusercontent.com/16770566/113624897-d6937b00-967d-11eb-94fc-3c61ae52a7a9.mov

@shahrukh-alam Was this the only action possible with the left bar buttons in both editors (gutenberg and classic) except from the site switcher? Will we be porting this functionality to navigation bar title now and making it a button?

**Comment by shahrukhalam** [comment]
@ceyhun 
> Was this the only action possible with the left bar buttons in both editors (gutenberg and classic) except from the site switcher?

I think that's right, we would have just the `close` button as the left bar button

> Will we be porting this functionality to navigation bar title now and making it a button?

We would have them as **Views** except the `Media Uploading`(used only on _classic editor_), which would remain a _button_

**Comment by shahrukhalam** [comment]
@iamthomasbishop 

> We could go larger, but let's just see how the current font-size (14pt) looks in relation to the Navbar Title.

|Before|After|
|---|---|
|Simulator Screen Shot - iPhone 12 Pro Max - 2021-04-07 at 13 31 21|Simulator Screen Shot - iPhone 12 Pro Max - 2021-04-07 at 13 39 51|

**Comment by iamthomasbishop** [comment]
@shahrukhalam That looks fine. Would you mind dropping a video/GIF that demonstrates the transition between the navbar title and "generating" message, for an in-context comparison? I just want to make sure the difference in font-size and change in font-weight don't make the text feel jumpy.

**Comment by shahrukhalam** [comment]
> Would you mind dropping a video/GIF that demonstrates the transition between the navbar title and "generating" message, for an in-context comparison?

@iamthomasbishop Please have a look 👇 

https://user-images.githubusercontent.com/16770566/113892463-a239cf00-97e3-11eb-87db-068d933d5643.mov

**Comment by iamthomasbishop** [comment]
That should work 👍 After @ceyhun has given his approval on the code side, feel free to ship it!

**Comment by shahrukhalam** [comment]
@ceyhun Pushed the changes as per the latest design at 8f91ffc & 70c712c
Please have a 👀

File: WordPress/Classes/ViewRelated/Views/WPBlogSelectorButton.m (hunk 2 above)
Comment (ceyhun) [reply WordPress/Classes/ViewRelated/Views/WPBlogSelectorButton.m:23]: > I think it is 😄

I'm still not sure why this is related to this PR. I think the `WPBlogSelectorButton` will become unused when this PR is merged. So why are we modifying it?

> I think this line was not really needed, it was getting ignored by the OS as AccessibilityLabel of barButtonItem was sufficient for the user.

When I try it on current `develop` branch it is not ignored. I tested on iOS 14.4 with an iPhone 11 Pro device. In which case was it ignored for you?

File: WordPress/Classes/ViewRelated/Views/WPUploadStatusButton.m [hunk 4]
```diff
@@ -14,7 +14,7 @@ - (instancetype)initWithFrame:(CGRect)frame
     }
     self.frame = frame;
     self.titleLabel.textColor = [UIColor whiteColor];
-    self.titleLabel.font = [WPFontManager systemBoldFontOfSize:14.0];
+    self.titleLabel.font = [WPFontManager systemRegularFontOfSize:14.0];
```
Comment (ceyhun) [inline WordPress/Classes/ViewRelated/Views/WPUploadStatusButton.m:17]: Nice to update this in classic editor as well! But I think the text is again not visible in light mode. Also it seems like the title view doesn't go back to site name after the upload is completed.

https://user-images.githubusercontent.com/1845482/114214812-1a7bce00-9965-11eb-98fb-0243f99ddb87.mp4

File: RELEASE-NOTES.txt [hunk 5]
```diff
@@ -3,6 +3,7 @@
 
 * [*] Reordered categories in page layout picker [#16156]
 * [*] Added preview device mode selector in the page layout previews [#16141]
+* [**] Removed Site Switcher in the Editor
```
Comment (ceyhun) [inline RELEASE-NOTES.txt]: Let's move this to 17.2 as 17.1 is already code freezed.

File: WordPress/Classes/ViewRelated/Views/LoadingStatusView.swift [hunk 6]
```diff
@@ -15,8 +15,8 @@ class LoadingStatusView: UIView {
```
Comment (ceyhun) [inline WordPress/Classes/ViewRelated/Views/LoadingStatusView.swift:13]: Seems to me like the `accessibilityHint` is not relevant here anymore. Also seems like we are passing in an already localized string for the title so maybe re-localizing it is not necessary with localizedStringWithFormat? What do you think?

**Review by ceyhun — COMMENTED** [review]
Thanks for the updates @shahrukhalam! Overall this looks and works great! Just have a few comments around accessibility and a small issue with the classic editor.

**Comment by ceyhun** [comment]
A small note: If the PR definition includes a keyword followed by the issue number like `fixes #123`, the issue will [automatically be linked and closed](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) when the PR is merged.

**Comment by ceyhun** [comment]
I also noticed a few commits by @shahrukh-alam, I'm guessing that's also you 😄

File: WordPress/Classes/ViewRelated/Views/WPBlogSelectorButton.m (hunk 2 above)
Comment (shahrukhalam) [reply WordPress/Classes/ViewRelated/Views/WPBlogSelectorButton.m:23]: > I'm still not sure why this is related to this PR. I think the WPBlogSelectorButton will become unused when this PR is merged. So why are we modifying it?

Yeah, You are right 👍, will revert

> When I try it on current develop branch it is not ignored. I tested on iOS 14.4 with an iPhone 11 Pro device. In which case was it ignored for you?

My bad 😃, that I tested on simulator
Tried on Device now, it's working perfectly fine

File: WordPress/Classes/ViewRelated/Views/WPUploadStatusButton.m (hunk 4 above)
Comment (shahrukhalam) [reply WordPress/Classes/ViewRelated/Views/WPUploadStatusButton.m:17]: > But I think the text is again not visible in light mode

Fixed at 0d91b5d58272f9a3a20faca5fdc68679545e3183

> Also it seems like the title view doesn't go back to site name after the upload is completed.

Great catch, fixed at 12ed6438605b4580dd5ec94d33cfcfc7e4202e87

https://user-images.githubusercontent.com/16770566/114430781-67b5a500-9bdc-11eb-9c35-6873530e3d68.mov

File: WordPress/Classes/ViewRelated/Views/LoadingStatusView.swift (hunk 6 above)
Comment (shahrukhalam) [reply WordPress/Classes/ViewRelated/Views/LoadingStatusView.swift:13]: Good Catch 💯 
Fixed at dbe48ce31b609337a4b47c0dee88a9c093432b0d

File: WordPress/Classes/ViewRelated/Views/WPBlogSelectorButton.m (hunk 2 above)
Comment (shahrukhalam) [reply WordPress/Classes/ViewRelated/Views/WPBlogSelectorButton.m:23]: > Yeah, You are right 👍, will revert

Fixed at a9291203265780ea296548499797310ccd9d554d

File: RELEASE-NOTES.txt (hunk 5 above)
Comment (shahrukhalam) [reply RELEASE-NOTES.txt]: Is it just about merging after code freeze or do we need to do something specific.
BTW when does code freeze end for 1.7.1

**Comment by shahrukhalam** [comment]
> I also noticed a few commits by @shahrukh-alam, I'm guessing that's also you 😄

Yeah, that's me 😄

File: RELEASE-NOTES.txt (hunk 5 above)
Comment (ceyhun) [reply RELEASE-NOTES.txt]: The process is described here: https://mobile.blog/2019/05/06/wordpress-mobile-apps-the-heartbeat-release-process/
We don't need to do something specific for this case 🙂 

And all the related dates should be here: https://github.com/wordpress-mobile/WordPress-iOS/milestones

Comment (shahrukhalam) [reply RELEASE-NOTES.txt]: Thanks for sharing the Docs, very helpful 💯

**Comment by ceyhun** [comment]
Thanks for the updates @shahrukhalam! I think the changes look good and I've also tested on an iPhone 11 Pro device with iOS 14.4 and it works great! Only thing remaining is to update the release notes and make the CI pass. 

circle-ci-checks

Do you also see the CircleCI checks like this?

**Comment by shahrukhalam** [comment]
@ceyhun 

> Only thing remaining is to update the release notes and make the CI pass.

Moved the issue fix to 17.2 at 2e1c5d4

> Do you also see the CircleCI checks like this?

Yeah 😄, shouldn't it be picked automatically or Do we need some permission to trigger it manually.

Just checked the CircleCI config, `Build Tests` and `Unit Tests` are part of the work flow & there is no filter for only to some specific branches. May be it's not picking up on PRs coming from Forks, not sure 😕

**Comment by ceyhun** [comment]
> Just checked the CircleCI config, `Build Tests` and `Unit Tests` are part of the work flow & there is no filter for only to some specific branches. May be it's not picking up on PRs coming from Forks, not sure 😕

I think that's the case here. I'll try pushing this branch to the main repo 👍

**Review by ceyhun — APPROVED** [review]

**Pull request merged**